site/content/in-dev/unreleased/getting-started/_index.md

```diff
@@ -23,19 +23,105 @@ type: docs
 weight: 101
 ---
 
-There are several options for getting started with Apache Polaris.
+# Getting Started with Apache Polaris Binary Distribution
 
-To quickly try out Apache Polaris, please use the [quickstart guide](./quick-start). For other examples, please see below.
+Quickly start Apache Polaris by running the pre-built binary, no build needed.
 
-## Docker Compose Examples
-Each of the proceeding Docker Compose examples provides a complete working environment with detailed instructions.
+---
+
+## Prerequisites
+
+- Java 21 or later installed. You can verify this by running:
+
+```bash
+java -version
+```
+
+---
+
+## Step 1: Download the Apache Polaris Binary
+
+1. Visit the official Apache Polaris GitHub
+   [Releases page](https://github.com/apache/polaris/releases).
+
+2. Download the latest binary archive file, for example:
+
+```bash
+curl -L https://downloads.apache.org/incubator/polaris/1.0.0-incubating/polaris-bin-1.0.0-incubating.tgz | tar xz
+```
+
+---
+
+## Step 2: Extract the Archive
+
+Extract the downloaded tar.gz file to your desired directory:
+
+```bash
+cd apache-polaris-1.0.0-incubating-bin
+```
+
+---
+
+## Step 3: Configure Polaris (Optional)
+
+Configure Polaris using environment variables or system properties. For example:
+
+```bash
+export POLARIS_JAVA_OPTS="-Dpolaris.storage.backend=local -Dpolaris.storage.local.path=/data/polaris"
+```
+
+Alternatively, you can set options when starting the server in Step 4.
+
+---
+
+## Step 4: Run the Polaris Server
 
-### Next Steps
+Start the Polaris server:
 
-1. Check & install dependencies
-2. Choose the way you want to deploy Polaris
-3. Create a catalog
-4. Check Using Polaris page
+```bash
+bin/server
+```
+
+The server will start on `http://localhost:8181` by default.
+
+---
+
+## Step 5: Verify the Server is Running
+
+Open your browser and navigate to:
+
+```bash
+curl http://localhost:8181/api/catalog/v1/health
+```
+
+You should see the Polaris server running or be able to access its REST API.
+
+---
+
+## Step 6: Stop the Polaris Server
+
+To stop the server, use `Ctrl+C` in the terminal where it's running, or kill the process:
+
+```bash
+pkill -f "java.*quarkus-run.jar"
+```
+
+---
+
+## Additional Resources
+
+- See the [official Apache Polaris documentation](https://polaris.apache.org/docs/) for comprehensive information on configuration, deployment, and usage.
+- Use `bin/admin` in the binary distribution for administrative and maintenance tasks (e.g., `bin/admin bootstrap`, `bin/admin purge`).
```

Comment:
I'd suggest to take a look at https://github.com/apache/polaris/blob/main/runtime/distribution/README.md for more context.